From ad2064dbb457482f9af9816f61ba721cc5aae126 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcin=20=C5=9Alusarz?= <marcin.slusarz@intel.com>
Date: Wed, 26 Feb 2020 15:54:19 +0100
Subject: [PATCH] test: build obj_defrag_advanced with some optimizations

On my machine it decreases TEST3 time from 52s to 20s
and TEST5 from 30s to 11s.
---
 src/test/obj_defrag_advanced/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/test/obj_defrag_advanced/Makefile b/src/test/obj_defrag_advanced/Makefile
index f2e8eda94..45fa851aa 100644
--- a/src/test/obj_defrag_advanced/Makefile
+++ b/src/test/obj_defrag_advanced/Makefile
@@ -40,4 +40,4 @@ LIBPMEMOBJ=y
 
 include ../Makefile.inc
 
-CFLAGS += -DDEBUG
+CFLAGS += -DDEBUG -Og
-- 
2.20.1

